high-availability.helmfile.yaml

```diff
@@ -11,10 +13,26 @@ releases:
     namespace: dapr-system
     createNamespace: true
     chart: dapr/dapr
-    version: 1.11.1
+    version: 1.13.2
     wait: true
+  - name: gatekeeper
```

Comment:
curious why we didn't add GK in the past.

Reply:
yeah this was an oversight. The idea originally was that a user would use helmfile to initially install and the HA helmfile to upgrade to HA so GK was assumed to be present. I don't think this is a valid assumption anymore.